packages/asset-service/src/generateAssetData/Utils.test.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseAsset } from '@shapeshiftoss/types'
+
+import { BTCMockedAsset, ETHMockedAsset } from '../service/AssetServiceTestData'
+import blacklist from './blacklist.json'
+import { filterBlacklistedAssets } from './utils'
+
+const blacklistedAssets: string[] = blacklist
+
+jest.mock('./blacklist.json', () => [ETHMockedAsset.tokens![0].caip19, BTCMockedAsset.caip19], {
+  virtual: true
+})
+
+const assetList: BaseAsset[] = [
+  Object.assign({}, ETHMockedAsset),
+  Object.assign({}, BTCMockedAsset)
+]
+
+describe('Utils', () => {
+  describe('filterBlacklistedAssets', () => {
+    it('should filter BTC from the asset list', () => {
+      const filteredAssetList = filterBlacklistedAssets(blacklistedAssets, assetList)
+      expect(filteredAssetList[0]).toHaveProperty('caip19', ETHMockedAsset.caip19)
+    })
+
+    it('should filter ERC20 from the asset list', () => {
+      const filteredAssetList = filterBlacklistedAssets(blacklistedAssets, assetList)
+      const ethFiltered = filteredAssetList[0]
+      const blacklistedToken = ETHMockedAsset.tokens![1]
```

Comment:
I have the same problem here, we know that tokens exist, and it's not related to the test goal at all

Reply:
fixed using `as const` in mocks